mesonbuild/compilers/mixins/emscripten.py

```diff
@@ -78,9 +78,8 @@ def find_library(self, libname: str, env: 'Environment', extra_dirs: T.List[str]
                      libtype: LibType = LibType.PREFER_SHARED, lib_prefix_warning: bool = True) -> T.Optional[T.List[str]]:
         if not libname.endswith('.js'):
             return super().find_library(libname, env, extra_dirs, libtype, lib_prefix_warning)
-        if os.path.isabs(libname):
-            if os.path.exists(libname):
-                return [libname]
+        if os.path.isabs(libname) and os.path.exists(libname):
+            return [libname]
```

Comment:
> make the intent easier to understand with fewer lines of code (-77 lines in this PR).

Simply reducing the number of lines is not the same as making the intent easier to understand. For an example, python supports rewriting this:

```suggestion
        if os.path.isabs(libname) and os.path.exists(libname): return [libname]
```

But most people wouldn't do that.

Reply:
Let's not swim upstream against PEP8.